github/teams_memebers_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
```

Comment:
minor type: Rename this to `team_members_test.go`

Reply:
I will fix `memebers`, but actual service name is `teams`. Shouldn't the file name start with `teams` then?